tests/examples/ShoutCaseEnumMembersLinter/prefer_require_once_linter.php.autofix.expect

```diff
@@ -0,0 +1,34 @@
+<?hh
+
+enum EmptyEnum: int {}
+
+enum GoodEnum: int {
+  GOOD_MEMBER = 1;
+  ALSO_GOOD_MEMBER = 1;
+}
+
+enum BadEnum: int {
+  CAMEL_CASE = 1;
+  SHOUT_CASE = 1;
+  SNAKE_CASE = 1;
+  CAMEL_CASE_WITH_ALOT_OF_WORDS = 1;
+  SNAKE_CASE_WITH_A_LOT_OF_WORDS = 1;
+  A_NUMBER_4_IN_THE_MIDDLE_OF_2_WORDS = 1;
+  A_NUMBER_AT_THE_END_3 = 1;
+  // I don't break out HTML and Instead, but we could.
+  GET_SOME_HTMLINSTEAD = 1;
+}
+
+enum CollidingNamesEnum: int {
+  CAMEL_CASE = 1;
+  // Really pascal case
+  CAMEL_CASE = 1;
```

Comment:
Looks like the auto-fix caused duplicate names here, is `isFixableWithoutCollisions` not working correctly?

Reply:
Good catch. If renaming two independent members causes a conflict, this goes wrong.
camelCase and CamelCase both become CAMEL_CASE.
During the time when I check, the old names are always there.
This causes a strange situation where the protection only works when the name was already in use in the original enum.
If CAMEL_CASE is already taken before the lint run started, it works like it should.
How do I force hhast to reload the affected files?